### PR TITLE
Configured Prettier and Github Actions

### PR DESCRIPTION
**Pull request by MathyouMB**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "overrides": [
+      {
+        "files": ["*.html"],
```
Comment (AngelOnFira) [inline .prettierrc:4]: @MathyouMB are you not getting it to format JS as well? It does it pretty well imo

Comment (MathyouMB) [reply .prettierrc:4]: It does format the js and css, but it uses the default config.

Comment (MathyouMB) [reply .prettierrc:4]: The config I added is specifically for working with the go templating syntax.

Comment (AngelOnFira) [reply .prettierrc:4]: Ah I see

**Pull request merged**